.openpublishing.redirection.json

```diff
@@ -50936,14 +50936,134 @@
         "redirect_document_id": false
     },
     {
-      "source_path": "docs-ref-services/preview/appinsights.md",
-      "redirect_url": "/azure/azure-monitor/app/java-in-process-agent",
-      "redirect_document_id": false
+        "source_path": "docs-ref-services/preview/appinsights.md",
+        "redirect_url": "/azure/azure-monitor/app/java-in-process-agent",
+        "redirect_document_id": false
     },
     {
-      "source_path": "docs-ref-services/latest/batchai.md ",
-      "redirect_url": "https://aka.ms/batchai-retirement",
-      "redirect_document_id": true
+        "source_path": "docs-ref-services/latest/batchai.md ",
+        "redirect_url": "https://aka.ms/batchai-retirement",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/batch-ai.md ",
+        "redirect_url": "https://aka.ms/batchai-retirement",
+        "redirect_document_id": true
+    },    
+    {
+        "source_path": "docs-ref-services/latest/appservice.md",
+        "redirect_url": "/java/api/overview/azure/app-service?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/cdn.md",
+        "redirect_url": "/java/api/overview/azure/content-delivery-network?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/cosmosdb.md",
+        "redirect_url": "/java/api/overview/azure/cosmos-db?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/datalakeanalytics.md",
+        "redirect_url": "/java/api/overview/azure/data-lake-analytics?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/entitysearch.md",
+        "redirect_url": "/java/api/overview/azure/entity-search?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/eventgrid.md",
+        "redirect_url": "/java/api/overview/azure/event-grid?view=azure-java-stable",
+        "redirect_document_id": true
+    },
+    {
+        "source_path": "docs-ref-services/latest/eventhubs.md",
```

Comment:
Redirect for Event Hubs should start from page originally linked in the ToC: eventhub.md

The original file was `eventhub.md` and that's the file that will return a 404 if someone tries to visit the URL directly. https://github.com/Azure/azure-docs-sdk-java/pull/2189/files#diff-cfc28a9b62a22a9af73713ec6f6de0c6a283917fca7a6ca2f7a54927dd68068a